.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+name: Run tests
+on:
+  push:
+  pull_request:
+    branches: [ master ]
+jobs:
+    test:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [2.7, 3.6, 3.7, 3.8, 3.9, "3.10"]
```

Comment:
The quotes made me look at the docs, and I think perhaps we should be doing all of them in quotes:
[docs](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-multiple-python-versions)